.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 💡 General questions
+    url: https://discord.com/invite/feTf9x3ZSB
```

Comment:
Cool - I just tried and the invite link works as well if you're already in the discord.

Thanks!